apps/admin-gui/src/app/users/pages/user-detail-page/user-accounts/user-accounts.component.html

```diff
@@ -0,0 +1,88 @@
+<h1 class="page-title">{{'USER_DETAIL.ACCOUNTS.TITLE' | translate}}</h1>
+<mat-spinner *ngIf="initLoading"></mat-spinner>
+<perun-web-apps-vo-search-select
+  *ngIf="!initLoading"
+  [vos]="vos"
+  (voSelected)="loadMember($event)">
+</perun-web-apps-vo-search-select>
+<mat-tab-group>
+  <mat-tab>
+    <ng-template matTabLabel>
+      {{'USER_DETAIL.ACCOUNTS.STATUS' | translate}}
+    </ng-template>
+    <ng-template matTabContent>
+      <mat-spinner *ngIf="loading"></mat-spinner>
+      <div *ngIf="!loading">
+        <div>
+          {{'USER_DETAIL.ACCOUNTS.MEMBER' | translate}}:
+          <a class="member-link" [routerLink]="['/organizations', selectedVo.id, 'members', member.id]"
+             queryParamsHandling="merge">
+            {{member.id}}
+          </a>
+        </div>
+        <div class="align-cards">
+          <app-member-overview-membership
+            [member]="member"
+            [vo]="selectedVo">
+          </app-member-overview-membership>
+
+          <app-member-overview-groups
+            [member]="member"
+            [vo]="selectedVo">
+          </app-member-overview-groups>
+        </div>
+      </div>
+    </ng-template>
+  </mat-tab>
+
+  <mat-tab>
+    <ng-template matTabLabel>
+      {{'USER_DETAIL.ACCOUNTS.GROUPS' | translate}}
+    </ng-template>
+    <ng-template matTabContent>
+      <mat-spinner *ngIf="loading"></mat-spinner>
+      <perun-web-apps-groups-list
+        *ngIf="!loading"
+        [groups]="groups" [displayedColumns]="['id', 'recent', 'name', 'description']">
+      </perun-web-apps-groups-list>
+    </ng-template>
+  </mat-tab>
+
+  <mat-tab>
+    <ng-template matTabLabel>
+      {{'MEMBER_DETAIL.ENTITY' | translate}}
+    </ng-template>
+    <ng-template matTabContent>
+      <mat-spinner *ngIf="loading"></mat-spinner>
+      <app-one-entity-attribute-page *ngIf="!loading" [entity]="'member'" [entityId]="member.id">
```

Comment:
I think there could be fine some header "Attributes", because the tab has only name "Member", but then is shown only attrubute list, so with this header it can be more clear.

Reply:
Header changed